tests/test_crewai_adapter.py

```diff
@@ -364,3 +364,133 @@ def test_none_values_filtered_from_kwargs(mcp_server_that_rejects_none_script):
         assert "Another: None" in result
         # Most importantly, no error message about "is not of type string, is <nil>"
         assert "parameter" not in result or "is <nil>" not in result
+
+
+@pytest.fixture
+def single_content_server_script():
```

Comment:
same here I would only keep the multiple content, the rest is already tested above

Reply:
👍